### ISSUE TITLE
Uncaught ReferenceError

### ISSUE THREAD

**Issue by ghigo** [pvvx/ATC_MiThermometer#387]
The webpage console shows the following error

TelinkMiFlasher.html:1139 Uncaught ReferenceError: hwver_id is not defined
    at menuUpgrade (TelinkMiFlasher.html:1139:44)

Unable to flash a build

**Comment by Danstiv** [comment]
Same problem on Telink Flasher v10.1
Configuration fields and firmwares are not showing.
What can I do with this?
```
01:07:37: Searching for devices
01:08:00: Connecting to: ATC_6C557D
01:08:06: Hardware Revision String: B1.5
01:08:06: Software Revision String: V4.7
01:08:06: Firmware Revision String: github.com/pvvx
01:08:06: Detected custom Firmware
```
@pvvx

**Comment by pvvx** [comment]
Found:
```
TelinkMiFlasher.html:847 Uncaught ReferenceError: hwver_id is not defined
```
Fixed in v10.2.
